src/main/java/ch/njol/skript/expressions/ExprParseError.java

```diff
@@ -27,23 +30,34 @@
 		"		message \"&lt;red&gt;Please put an integer on line 1!\""})
 @Since("2.0")
 public class ExprParseError extends SimpleExpression<String> {
+
 	static {
-		Skript.registerExpression(ExprParseError.class, String.class, ExpressionType.SIMPLE, "[the] [last] [parse] error");
+		Skript.registerExpression(ExprParseError.class, String.class, ExpressionType.SIMPLE, "[the] [last] [parse] error[all:s]");
 	}
+
+	private static final List<String> lastErrors = new ArrayList<>();
```

Comment:
I wonder if we want this to be based on Events/Contexts rather than be global across all triggers. Maybe out of scope for this PR though